js/libs/quantimodo-api.js

```diff
@@ -73,14 +73,14 @@ Quantimodo = function () {
                 url: hostUrl + baseURL,
                 contentType: 'application/json',
                 beforeSend: function (xhr) {
-                    xhr.setRequestHeader("Authorization", "Bearer " + access_token);
+                    xhr.setRequestHeader('Authorization', 'Bearer ' + access_token);
```

Comment:
Line is too long.<br>Identifier 'access_token' is not in camel case.<br>'access_token' is not defined.